Node/core/src/bots/ChatConnector.ts

```diff
@@ -549,6 +549,7 @@ export class ChatConnector implements IConnector, IBotStorage {
 
     private dispatch(msg: IMessage, res: IWebResponse, next: Function) {
```

Comment:
`private dispatch(msg: IMessage, res: IWebResponse, next = () => void) {` not sure if it does  the same

Reply:
I haven't tried that but it is an interesting item. I'm not sure an arrow function would work in this context, it would be interesting to see what `tsc` compiles it to.

Reply:
Yep, I was testing that because I wasn't sure either. Looks like you'd need to change the other signatures from `next: Function` to `next: () => void` and here use `next = () => {}`

It generates this `if (next === void 0) { next = function () { }; }`

Reply:
Thanks for the feedback and info! As indicated I closed this PR due to Steve adding some changes to a different method which addresses the same problem.